internal/server/debug.go

```diff
@@ -22,10 +22,21 @@ var pprofRoute = route[api.EmptyRequest, *api.EmptyResponse]{
 	},
 }
 
-func pprofHandler(c *gin.Context, _ *api.EmptyRequest) (*api.EmptyResponse, error) {
+type pprofRequest struct{}
+
+func (pprofRequest) IsBlockingRequest() bool {
```

Comment:
This method is used in `wrapRoute` to indicate the request should get a longer timeout (5 minutes). It will allow us to get profile data for a longer period of time instead of the 1 minute timeout for regular requests.